src/core/net/ip6.cpp

```diff
@@ -59,6 +60,9 @@ Ip6::Ip6(Instance &aInstance)
     , mIcmp(aInstance)
     , mUdp(aInstance)
     , mMpl(aInstance)
+#if OPENTHREAD_ENABLE_IP6_FRAGMENTATION
+    , mTimer(aInstance, &Ip6::HandleTimer, this)
+#endif // OPENTHREAD_ENABLE_IP6_FRAGMENTATION
```

Comment:
nit: can remove `// OPENTHREAD_ENABLE_IP6_FRAGMENTATION` given the small block size.

Reply:
removed.